src/jimgw/single_event/likelihood.py

```
@@ -659,18 +694,22 @@ def phase_marginalized_likelihood(
     detectors: list[Detector],
     freqs: Float[Array, " n_dim"],
+    datas: list[Float[Array, " n_dim"]],
+    psds: list[Float[Array, " n_dim"]],
     align_time: Float,
     **kwargs,
 ) -> Float:
     log_likelihood = 0.0
     complex_d_inner_h = 0.0
     df = freqs[1] - freqs[0]
-    for detector in detectors:
+    f_min = freqs[0]
```

Ruff (0.8.2)

Local variable `f_min` is assigned to but never used

Remove assignment to unused variable `f_min`

(F841)